### PR TITLE
Update model function names

### PR DESCRIPTION
**Pull request by amanda-minter**
Function names in `{epidemics}` are now `model_X()` following this [PR ](https://github.com/epiverse-trace/epidemics/pull/135)

### CODE REVIEW

**Comment by avallecam** [comment]
I will need more time to look into this. Is one week viable for you or before?

**Comment by amanda-minter** [comment]
Please take a look as soon as you can, this is high priority as it affects the live site examples and is a blocker for @CarmenTamayo to review the materials.

File: episodes/model-choices.Rmd [hunk 1]
```diff
@@ -67,7 +67,7 @@ Finally, interventions such as vaccination may be of interest. A model may or ma
 ### Available  models
  
 The R package `epidemics` contains functions to run existing models.
-For details on the models that are available, see the package [vignettes](https://epiverse-trace.github.io/epidemics/articles). To learn how to run the models in R, read the documentation using `?epidemics::epidemic_ebola`. Remember to use the 'Check model equation' questions to help your understanding of an  existing model. 
+For details on the models that are available, see the package [vignettes](https://epiverse-trace.github.io/epidemics/articles). To learn how to run the models in R, read the documentation using `?epidemics::model_ebola_cpp`. Remember to use the 'Check model equation' questions to help your understanding of an  existing model.
```
Comment (avallecam) [inline episodes/model-choices.Rmd:70]:
```suggestion
For details on the models that are available, see the package [vignettes](https://epiverse-trace.github.io/epidemics/articles). To learn how to run the models in R, read the documentation using `?epidemics::model_ebola_r`. Remember to use the 'Check model equation' questions to help your understanding of an  existing model. 
```

Comment (avallecam) [reply episodes/model-choices.Rmd:70]: I think is `epidemics::model_ebola_r()` the one that leads to the reference manual. Now `model_ebola_cpp` is still in that reference manual, but as [an internal function not exported to NAMESPACE](https://stackoverflow.com/a/2675532/6702544), and to access it I need to use three double-colon like `epidemics:::model_ebola_cpp()`

Comment (avallecam) [inline episodes/model-choices.Rmd:70]:
```suggestion
For details on the models that are available, see the package [vignettes](https://epiverse-trace.github.io/epidemics/articles). To learn how to run the models in R, read the documentation using `?epidemics::model_ebola_cpp`. Remember to use the questions in the '[Check model equation](#check-model-equations)' checklist to help your understanding of an existing model. 
```

Comment (avallecam) [reply episodes/model-choices.Rmd:70]: Is this the checklist of questions are you referring to?

File: episodes/model-choices.Rmd [hunk 2]
```diff
@@ -90,9 +90,9 @@ You have been asked to explore the variation in numbers of infected individuals
 
 Which of the following models would be an appropriate choice for this task:
 
-+  `epidemic_default`
++  `model_default_cpp()`
 
-+ `epidemic_ebola`
++ `model_ebola_cpp()`
```
Comment (avallecam) [inline episodes/model-choices.Rmd:95]:
```suggestion
+ `model_ebola_r()`
```

File: episodes/model-choices.Rmd [hunk 3]
```diff
@@ -157,7 +157,7 @@ DiagrammeR::grViz("digraph {
 The model is capable of predicting an Ebola type outbreak, but as the model is deterministic, we are not able to explore stochastic variation in the early stages of the outbreak.
 
 
-#### `epidemic_ebola`
+#### `model_ebola_cpp()`
```
Comment (avallecam) [inline episodes/model-choices.Rmd:160]:
```suggestion
#### `model_ebola_r()`
```

**Review by avallecam — CHANGES_REQUESTED** [review]
Is the replacement of `model_ebola_r` instead of the `model_ebola_cpp` is coherent?

From the vignette, [in the "Run epidemic model" subtitle](https://epiverse-trace.github.io/epidemics/articles/ebola_model.html#run-epidemic-model):

> We run the model using the function [model_ebola_r()](https://epiverse-trace.github.io/epidemics/reference/epidemic_ebola.html). While there is a C++ equivalent of this model, it was not found to be much faster than the R-only version, and is not made available to users.

**Comment by avallecam** [comment]
I did the step of "Update with rebase" here on GitHub. So if you need to make new changes locally, probably you need to pull content first? Let me know if this generates any issues.

File: episodes/model-choices.Rmd (hunk 1 above)
Comment (amanda-minter) [reply episodes/model-choices.Rmd]: Yes, added your suggestion in this [commit](https://github.com/epiverse-trace/tutorials/commit/bcb9a3c5870523e736d642197646237dc6d34e2f) as the suggestion was outdated

**Review by avallecam — APPROVED** [review]

**Comment by avallecam** [comment]
I see a "branch cannot be rebased due to conflicts". I'm afraid that if it is not possible to resolve conflicts, then we should "squash and merge". I think I will avoid the "Update with rebase" option in GitHub, and we can do the rebasing in the local repos only by running [`git pull origin main`](https://docs.gitlab.com/ee/topics/git/git_rebase.html) to rebase the feature branch. I'll update on this in a separate issue/PR.

**Comment by amanda-minter** [comment]
> I see a "branch cannot be rebased due to conflicts". I'm afraid that if it is not possible to resolve conflicts, then we should "squash and merge" for this one, again.
> 
> I think I will avoid the "Update with rebase" option in GitHub, and we as developers can do the rebasing in the local repos only by running [`git pull origin main`](https://docs.gitlab.com/ee/topics/git/git_rebase.html) to rebase the feature branch. I'll update on this in a separate issue/PR.

Thanks Andree, I will have to squash and merge this PR

**Pull request merged**